components/pedigree/resources/src/main/resources/pedigree/view/cancersLegend.js

```diff
@@ -51,7 +51,7 @@ define([
                                    "Myeloma" : "Myeloma",
                                    "Leukemia" : "Leukemia"
                                  };
-            $super('Cancers', true);
```

Comment:
The rest of the changes don't seem to be related to PT-2825, are they? Can this commit be split in two?